lstchain/reco/utils.py

```diff
@@ -239,12 +239,9 @@ def camera_to_altaz(pos_x, pos_y, focal, pointing_alt, pointing_az, obstime = No
     sky_coords = utils.camera_to_altaz_frame(pos_x, pos_y, focal, pointing_alt, pointing_az)
 
     """
-
-    if obstime:
-        horizon_frame = AltAz(location=location, obstime=obstime)
-    else:
-        horizon_frame = AltAz(location=location, obstime=obstime_mc)
+    if not obstime:
         logging.warning("No time given, use default observation time. To be use only for MC data.")
```

Comment:
Now, you don't use any obstime (obstime is None, you are not using the global)

Reply:
yes this is what you ask no?

Reply:
But the logging message says something about a default obstime, you not using any obstime

Reply:
And I would say, as this is the expected situation for every simulated event, this should not be a "warning".

Warnings should be for unexpected conditions that require the user to do something. Meaningless warnings are dangerous because then users start ignoring them and potentially start ignoring also the important warnings.

Reply:
This is because you are too fast, I already modify the print in the time you did your comment

Reply:
I agree for the warning